lib/services/ngsi/ngsiService.js

```diff
@@ -494,6 +494,22 @@ function sendUpdateValueNgsi2(entityName, attributes, typeInformation, token, ca
                 delete payload.type;
                 options.json = payload;
             }
+            // Try to clean object_id from options.json
```

Comment:
Maybe it would be good to add a brief comment explaining why we have to clean object_id here.

Reply:
Fixed in a79ffd8438cadf1fd25a5cc22e3e49ec8170a437

Reply:
Fixed in a79ffd8438cadf1fd25a5cc22e3e49ec8170a437